Dockerfile

```diff
@@ -1,7 +1,20 @@
-FROM python:3.5
-WORKDIR /code
-ADD . /code
-RUN pip install cliquet[postgresql,monitoring]
-RUN pip install -e .
-RUN kinto --backend=memory init
-CMD kinto start
+# Mozilla Kinto server
+FROM stackbrew/debian:sid
```

Comment:
the size is virtual AFAIK

that means if I already have the python image, then I will only download the new layer of kinto. Whereas if the container is built from a raw debian I download everything